lib/game.rb

```diff
@@ -0,0 +1,92 @@
+require_relative '../lib/board.rb'
+
+class Game
+
+  attr_accessor :player_1, :player_2
+
+  WIN_COMBINATIONS= [
+      [0,1,2],  #top row
+      [3,4,5], #middle row
+      [6,7,8],  #bottom row
+      [0,3,6],  #left column
+      [1,4,7],  #middle column
+      [2,5,8],  #right column
+      [0,4,8],  #diagonal 1
+      [6,4,2]   #diagonal 2
+  ]
+
+  def initialize(player_1 =  Human.new('X'), player_2 = Human.new('O'), board = Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+  end
+
+  def cells=(set_cells)
+    @board.cells = set_cells
+  end
+
+  def cells
+    (@board.cells == []) ? @board.cells : @board
+  end
+
+  def won?
+    $winner = nil
+    returnable = []
+    WIN_COMBINATIONS.each do |win_combo|
+      if @board.cells[win_combo[0]] == 'X' && @board.cells[win_combo[1]] == 'X' && @board.cells[win_combo[2]] == 'X'
+        $winner = player_1.token
+        returnable = win_combo
+        break
+      elsif @board.cells[win_combo[0]] == 'O' && @board.cells[win_combo[1]] == 'O' && @board.cells[win_combo[2]] == 'O'
+        $winner = player_2.token
+        returnable = win_combo
+        break
+      else
+        returnable = false
+      end
+    end
+    returnable
```

Comment:
You should never `return` from within a loop. If you need a return value from within a loop you need to use a higher level iterator or some sort of local variable. 

Consider this solution to a simpler problem of finding the even number in an array.

```ruby
the_even_number = nil
[1,2,3].each do |i|
  if i.even?
    the_even_number = i
  end
end
the_even_number
```

Instead of returning from within that loop, we flip the switch local variable `the_even_number` to the element in the array that made the if statement true. Then we leave `the_even_number` as the last line to ensure the return value. If no even numbers are present, `the_even_number` will remain nil.

But better, let's just use [`Enumerable#detect`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-detect)

```ruby
the_even_number = [1,2,3].detect do |i|
  i.even?
end
```